Config/AltBuildSystems/TasmanianSG.py

```diff
@@ -39,7 +39,7 @@
     bTsgPlotting = False
     tsgPlot = []
 
-__version__ = "5.1"
+__version__ = "6.0"
```

Comment:
Should this be coming from the `C/C++`?

Reply:
All version info comes from the same line in cmake, there is also regression test that ensures that those will match.